string name
float64 ctrl_joint_angle
geometry_msgs/Pose pose
